app/assets/javascripts/application/evaluation.js

```diff
@@ -11,6 +11,37 @@ var evaluation = (function(Rx, $) {
 
             inputStream.subscribe(inputChange);
         }
+        $('.principle').click(function() {
+            principleModal($(this))
+        });
+    }
+
+    function principleModal (principle) {
```

Comment:
I don't think we should use ES6 template literals, surely not mixed like on line 21. It will just bring inconsistence into the project. Maybe later on we can use babel with full ES6 support, to avoid incompatibility with browsers.

On the other hand, I think it's nicer to create proper named vars and just use them during the text concatenating, instead of calling many functions in the text.